graylog2-web-interface/src/views/bindings.jsx

```diff
@@ -109,6 +109,8 @@ ViewSharing.registerSubtype(SpecificUsers.Type, SpecificUsers);
 Parameter.registerSubtype(ValueParameter.type, ValueParameter);
 Parameter.registerSubtype(LookupTableParameter.type, LookupTableParameter);
 
+const isAnalysisDisabled = (field: string, analysisDisabledFields: string[] = []) => analysisDisabledFields.includes(field);
```

Comment:
This function is very simple, but I would still prefer an own function to do the "analysis disabled" check. I would like to avoid, adding helper functions to the `bindings` file. Were would you add this function? Maybe as a static method of `views/logic/fieldtypes/FieldType.js`?

Reply:
Regarding the default value for  `analysisDisabledFields`. Currently the value can't be `null` or `undefined`, but I saw no disadvantage in adding the default value. It also makes testing of related code easier.